src/windows/AppVersionProxy.js

```diff
@@ -0,0 +1,19 @@
+AppVersionProxy = {
+  getVersionNumber: function (successCallback, failCallback, args) {
```

Comment:
I'm using this project template https://github.com/apache/cordova-windows running on Windows Phone 8.1 Update and the above method `getAppName`, works only with my implementation